docs/New-Maintainer-Checklist.md

```diff
@@ -14,13 +14,13 @@ but we'd appreciate your continuing help in keeping on top of contributions.
 
 A few requests:
 
-- please make pull requests on any changes to core (i.e. non-formula) code or
-  any non-trivial (e.g. not a test or audit improvement or version bump)
-  changes to formulae code and don't merge them unless you get at least one +1
-- use `brew pull` and let it auto-close issues wherever possible (it may take
-  ~5m). When this isn't possible always use `git pull --rebase`, `git rebase`
-  and `git cherry-pick` rather than `git merge` and never use GitHub's "Merge
-  pull request" button. If in doubt, check with GitX that you've not
+- please make pull requests on any changes to Homebrew/brew code or any
+  non-trivial (e.g. not a test or audit improvement or version bump) changes
+  to formulae code and don't merge them unless you get at least one approval
+  and passing tests.
+- use `brew pull` for formulae changes that require new bottles and let it
+  auto-close issues wherever possible (it may take ~5m). When this isn't necessary
+  use GitHub's "Merge pull request" button. If in doubt, check with GitX that you've not
```

Comment:
I know `Merge pull request` is used for brew, but I thought rebasing / cherry-picking was preferred for formulae?  https://github.com/Homebrew/brew/issues/1229#issuecomment-252292784